lerna.json

```diff
@@ -1,4 +1,4 @@
 {
   "$schema": "node_modules/lerna/schemas/lerna-schema.json",
-  "version": "16.6.1"
+  "version": "16.6.2"
```

Comment:
Make sure to rebase so that the git history is clean.

Reply:
The "publish" commit is still on the branch. Shall we look together into rebase?

Reply:
Rabased 🥳